main_test.go

```diff
@@ -15,6 +15,16 @@ func TestBaseCommandsAreRegistered(t *testing.T) {
 	baseManager := cmd.BuildBaseManager("tranor", "", "", nil)
 	manager := buildManager("tranor")
 	for name, expectedCommand := range baseManager.Commands {
+		var skip bool
+		for _, c := range baseCommandsToRemove {
```

Comment:
Since target-set is not removed, we should actually use the same list of removed commands we use below, no? Then we wouldn't even need the `TestTargetSetIsRegistered` test